app/javascript/App/components/NewsfeedPage/NewsList.js

```diff
@@ -20,6 +20,8 @@ class NewsList extends Component {
       this.setState({ initialRender: false })
     }, 60000)
     this.mountOnScrollHandler()
+    const newsfeedElem = document.querySelector('#newsfeed')
+    newsfeedElem.style.height = `${newsfeedElem.offsetHeight}px`
```

Comment:
@cryptoB0T This still doesn't work for me... the scrollbar goes past the screen and users don't see the spinning loading icon when loading the infinite scroll.

Reply:
@hanchang I see what you mean. as the user scrolls down, they page loads more items and the height of the container increases. let me think about this, I've got to get this to work..